src/Detail/components/LecueNoteListContainer/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+
+import LecueNoteListHeader from '../LecueNoteLIstHeader';
+import ZigZagView from '../ZigZagView';
+import * as S from './LecueNoteListContainer.style';
+
+interface Note {
+  noteId: number;
+  renderType: number;
+  content: string;
+  noteDate: string;
+  noteNickname: string;
+  noteBackgroundColor: number;
+}
+
+interface LecueNoteListContainerProps {
+  noteNum: number;
+  backgroundColor: number;
+  noteList: Note[];
+}
+
+function LecueNoteListContainer({
+  noteNum,
+  backgroundColor,
+  noteList,
+}: LecueNoteListContainerProps) {
+  const [isZigZagView, setIsZigZagView] = useState<boolean>(true);
+  return (
+    <S.LecueNoteListContainerWrapper backgroundColor={backgroundColor}>
+      <LecueNoteListHeader
+        noteNum={noteNum}
+        backgroundColor={backgroundColor}
+        isZigZagView={isZigZagView}
+        buttonOnClick={() => setIsZigZagView(!isZigZagView)}
+      />
+      {isZigZagView ? (
+        <ZigZagView noteList={noteList} />
+      ) : (
+        <ZigZagView noteList={noteList} />
+      )}
```

Comment:
ASK: 오잉 이렇게 되면 isZigZagView의 값에 관계없이 항상 `<ZigZagView noteList={noteList} />`가 실행되는 거 아닌가요?

Reply:
P1: 이 로직이 의도한 부분이라면 아래처럼 고치는게 좋을 것 같습니당!
```suggestion
      {isZigZagView && <ZigZagView noteList={noteList} />}
```

Reply:
아하 요것은 : 뒤에 일자뷰를 렌더링해야 하는데 아직 개발 전이라 둘 다 지그재그뷰를 렌더링하도록 임시로 만들었습니닷!!